### PR TITLE
Handle templates by php version

### PR DESCRIPTION
**Pull request by dantleech**

### CODE REVIEW

File: doc/configuration.md [hunk 1]
```diff
@@ -51,6 +51,13 @@ Reference
 
 ### Core
 
+#### php.version
+
+*Default*: `PHP_VERSION`
```
Comment (dantleech) [inline doc/configuration.md]: => `your current PHP version, e.g. `7.3`

File: doc/configuration.md [hunk 2]
```diff
@@ -51,6 +51,13 @@ Reference
 
 ### Core
 
+#### php.version
```
Comment (dantleech) [inline doc/configuration.md]: => `code.php_version` in general all config keys should be prefixed with their module name.

File: lib/Extension/Core/CoreExtension.php [hunk 3]
```diff
@@ -33,13 +33,15 @@ class CoreExtension implements Extension
...
     const COMMAND = 'command';
+    const CONFIG_PHP_VERSION = 'php.version';
 
     public function configure(Resolver $schema)
     {
         $schema->setDefaults([
             self::DUMPER => 'indented',
             self::XDEBUG_DISABLE => true,
             self::COMMAND => null,
+            self::CONFIG_PHP_VERSION => phpversion(),
```
Comment (dantleech) [inline lib/Extension/Core/CoreExtension.php]: => `PARAM_PHP_VERSION` (may as well rename the others too)

**Comment by elythyr** [comment]
Thanks for doing it yourself !

**Pull request merged**